nan_implementation_12_inl.h

```diff
@@ -337,7 +345,23 @@ Factory<v8::StringObject>::New(v8::Local<v8::String> value) {
 #if V8_MAJOR_VERSION >= 7
```

Comment:
At the other locations this `#if` was replaced by `#if NODE_MAJOR_VERSION >= 10`. Is it kept intentionally here?

Reply:
Yes. Should I add a comment?

Reply:
Up to you.
Personally I find the #ifs on V8 version not nice as it indicates that it breaks the API/ABI stability within a major nodejs version.

Reply:
Yeah, I agree. I've pushed a new commit that removes it, PTAL.